CMakeLists.txt

```diff
@@ -207,6 +207,11 @@ if(ENABLE_PARQUET)
   add_definitions(-DSUPP_PARQUET)
 endif()
 
+if(ENABLE_PLUGIN)
```

Comment:
check platform

Reply:
done, move the check in SUPP_PLUGIN to ENABLE_PLUGIN